config/config_test.go

```diff
@@ -213,6 +214,7 @@ func TestReload(t *testing.T) {
 	watcher := &configwatcher.ConfigWatcher{
 		Config: c,
 		PubSub: pubsub,
+		Logger: &logger.NullLogger{},
```

Comment:
is this change needed?

Reply:
Yes, because the test would panic if the logger is nil